src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -131,10 +131,15 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
                 funcName: "fluid.prefs.enactor.syllabification.parse",
                 args: ["{that}", "{arguments}.0"]
             },
+            createHyphenator: {
```

Comment:
Please add a comment in the "members" area defining "hyphenators" explaining that it is a map of Strings representing sources to Promises, etc.